Dockerfile

```diff
@@ -48,7 +51,7 @@ RUN curl -sSL https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/6.0.1/frees
     --exclude='freesurfer/trctrain'
 
 # Simulate SetUpFreeSurfer.sh
-ENV FSL_DIR="/usr/share/fsl/5.0" \
+ENV FSL_DIR="/opt/fsl-5.0.11" \
```

Comment:
Do we want FSL 6?

Reply:
we can try? i'm not sure how that migration would look tbh

Reply:
Okay, let's do it separately then. No need to complicate a fix.